conductor.json

```diff
@@ -323,20 +323,24 @@
     },
     {
       "name": "sqs",
-      "image": "localstack/localstack:0.11.2",
-      "port": 4576,
+      "image": "localstack/localstack:0.13",
+      "port": 4566,
       "env": {
         "SERVICES": "sqs",
-        "DEFAULT_REGION": "{{env.SQS_REGION}}"
+        "SQS_PROVIDER": "elasticmq"
       }
     },
     {
       "name": "aws-cli",
-      "image": "mesosphere/aws-cli",
+      "image": "amazon/aws-cli:2.4.6",
+      "depends_on": [
+        "sqs"
```

Comment:
Don't run until sqs is at least started (although it doesn't wait until it's running healthily annoyingly).